libnetwork/drivers/windows/overlay/overlay_windows.go

```diff
@@ -23,7 +23,6 @@ const (
 type driver struct {
 	config     map[string]interface{}
 	networks   networkTable
-	store      datastore.DataStore
 	localStore datastore.DataStore
```

Comment:
Somehow I'm not surprised that `store` wasn't actually used by the driver implementation. And just as unsurprisingly, neither is `localStore`. Mind cleaning that up too while you're at it?

Reply:
`once` and `joinOnce` too. I'm removing all of them :wink: